wearipedia/devices/fitbit/fitbit_sense_gen.py

```diff
@@ -0,0 +1,368 @@
+    }
+    minutesVeryActive = {"dateTime": date, "value": very_active}
+    minutesFairlyActive = {"dateTime": date, "value": fairly_active}
+    minutesLightlyActive = {"dateTime": date, "value": lightly_active}
+    distance = {
+        "dateTime": date,
+        "value": (very_active * 50 + fairly_active * 25 + lightly_active * 10) / 1326,
+    }
+    minutesSedentary = {
+        "dateTime": date,
+        "value": 1440 - (very_active + fairly_active + lightly_active),
+    }
+
+    return (
+        steps,
+        minutesVeryActive,
+        minutesFairlyActive,
+        minutesLightlyActive,
+        distance,
+        minutesSedentary,
+    )
+
+
+def get_heart_rate(date):
+    """Generate heart rate data for a given date.
+
+    :return: dictionary of heart rate values and details
+    :rtype: dictionary
+    """
+
+    heart_rate_data = {
+        "heart_rate_day": [
+            {
+                "activities-heart": [
+                    {
+                        "dateTime": date,
+                        "value": {
+                            "customHeartRateZones": [],
+                            "heartRateZones": [
+                                {
+                                    "caloriesOut": 2877.06579,
+                                    "max": 110,
+                                    "min": 30,
+                                    "minutes": 0,
+                                    "name": "Out of Range",
+                                },
+                                {
+                                    "caloriesOut": 0,
+                                    "max": 136,
+                                    "min": 110,
+                                    "minutes": 0,
+                                    "name": "Fat Burn",
+                                },
+                                {
+                                    "caloriesOut": 0,
+                                    "max": 169,
+                                    "min": 136,
+                                    "minutes": 0,
+                                    "name": "Cardio",
+                                },
+                                {
+                                    "caloriesOut": 0,
+                                    "max": 220,
+                                    "min": 169,
+                                    "minutes": 0,
+                                    "name": "Peak",
+                                },
+                            ],
+                            "restingHeartRate": 58,
+                        },
+                    }
+                ],
+                "activities-heart-intraday": {
+                    "dataset": [],
+                    "datasetInterval": 1,
+                    "datasetType": "minute",
+                },
+            }
+        ]
+    }
+
+    the_time = time(hour=0, minute=0, second=0)
+    for i in range(1440):
```

Comment:
1440 seems like a magic number, you should rename this to a variable so it is easy to tell what we are looping over. Also, as this appears to be the duration of the exercise, I think you should make it variable rather than fixed (set a random duration).